jest.config.js

```diff
@@ -7,9 +7,12 @@ module.exports = {
 	transformIgnorePatterns: [`/node_modules/(?!${esModules})`],
 	moduleNameMapper: {
 		'\\.[s]?css$': '<rootDir>/utils/storybook/styleMock.js',
+		'@pubpub/deposit-utils/datacite':
+			'<rootDir>/node_modules/@pubpub/deposit-utils/dist/cjs/datacite/index.js',
```

Comment:
Unfortunately had to add this alias because our version of Jest does not utilize the `exports` field in package.json when resolving package entrypoints.